src/css/_variables.scss

```diff
@@ -0,0 +1,19 @@
+
```

Comment:
Should we include a comment like the other files stating what's contained here?

Reply:
Good catch - Maybe we don't really need to, since it's evident in the filename. I don't have a preference either way as long as we keep it consistent.